qa-pipelines/tasks/lib/cf-deploy-upgrade-common.sh

```diff
@@ -54,13 +54,17 @@ UAA_ADMIN_CLIENT_SECRET="$(head -c32 /dev/urandom | base64)"
 is_namespace_ready() {
     local namespace="$1"
 
-    # Check that all pods which were not created by jobs are ready
-    [[ true == $(2>/dev/null kubectl get pods --namespace=${namespace} --output=custom-columns=':.status.containerStatuses[].name,:.status.containerStatuses[].ready' \
-        | grep -vE 'secret-generation|post-deployment' \
-        | awk '{ print $2 }' \
-        | sed '/^ *$/d' \
-        | sort \
-        | uniq) ]]
+    # Check that all pods not from jobs are ready
+    if ! kubectl get pods --namespace "${namespace}" --selector '!job-name' \
+        --output 'custom-columns=:.metadata.name,:.status.containerStatuses[].name,:.status.containerStatuses[].ready' \
+        | awk '{ if ($NF != "true") { print } }' \
+        | tee /dev/stderr \
```

Comment:
Just to remind me why this is stuck:
@HartS wants to drop the `tee` here, and instead manually print out what is wrong at the caller, so that we do not fill up logs with errors.

Reply:
Fixed in 86685d420c33614583d39909b44ca8577c2660b9 (plus actually look at _all_ containers).